instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -571,14 +571,14 @@ def __init__(
             self.duration_histogram_old = self.meter.create_histogram(
                 name=MetricInstruments.HTTP_SERVER_DURATION,
                 unit="ms",
-                description="measures the duration of the inbound HTTP request",
+                description="Measures the duration of inbound HTTP requests.",
```

Comment:
Not a big deal but looking at [v1.11.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.11.0/specification/metrics/semantic_conventions/http-metrics.md#http-server) the description seems correct. wdyt @lzchen ?

Reply:
I saw that too but the link to the OT specs migration on the website seems more "current" -> https://opentelemetry.io/docs/specs/semconv/non-normative/http-migration/#http-server-duration-metric. I don't have a strong opinion though.